src/components/RadioButton/RadioButton.js

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes as toBe, Component } from 'react';
+import './RadioButton.css';
+
+class RadioButton extends Component {
+  static propTypes = {
+    name: toBe.string.isRequired,
+    color: toBe.string.isRequired,
+    onClick: toBe.func.isRequired,
```

Comment:
I suggest renaming `onClick` to `onChange` because value of radio is changed and `onChange` semantically better